dfencoder/scalers.py

```diff
@@ -4,6 +4,14 @@
 import torch
 from sklearn.preprocessing import QuantileTransformer
 
+def ensure_float_type(x: typing.Union[torch.Tensor, np.ndarray]):
+    """Ensure we are in the right floating point format. """
+    if (isinstance(x, torch.Tensor)):
+        result = x.to(dtype=torch.float32, copy=True)
+    elif (isinstance(x, np.ndarray)):
+        result = x.astype(float)
+    return result
```

Comment:
Raise an exception if it is not one of these two types.

Reply:
Resolved when merging in branch-23.01
https://github.com/nv-morpheus/dfencoder/pull/6/commits/4bdfbe025d040d4d2059e2434001bfeeefc06916